testing/docker/bookinfo.yaml

```diff
@@ -33,6 +33,19 @@ gateway:
   container_name: gateway
   command: /usr/bin/run_filebeat.sh
 
+tcphelloworld:
+  image: amalgam8/a8-tcp:latest
+  environment:
+    - A8_REGISTRY_URL=http://registry:8080
+    - A8_SERVICE=tcphelloworld
```

Comment:
Please attach a tag to this service name.
e.g.
`A8_SERVICE=tcphelloworld:version=v1`